### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by hbertrand**
- Clarified what batchnorm is and where to use it
- Added a reminder to use Tensor.view before nn.Linear
- Typos

### CODE REVIEW

File: tutoriaux/CNN/CNN_a_completer.ipynb [hunk 1]
```diff
@@ -1200,21 +1201,20 @@
...
     "\n",
     "### Boîte à outils\n",
-    "Pour ajouter la batch normalisation dans LeNet5, il suffit de l'ajouter parmis les couches de l'`__init__`. La classe suivante peut être utilisée:\n",
+    "Pour ajouter la batch normalisation dans LeNet5, il suffit de l'ajouter parmis les couches de l'`__init__` et de l'appeler après chaque Relu. La classe suivante peut être utilisée:\n",
     "<ul>\n",
     "<li><a href=\"http://pytorch.org/docs/master/nn.html#torch.nn.BatchNorm2d\">`nn.BatchNorm2d(num_features)`</a> : permet d'ajouter de la batch normalisation à une entrée à 4 dimensions présentée sous la forme d'un tenseur à 3 dimensions.</li>\n",
     "</ul>\n",
     "\n",
-    "### Implémentation\n",
-    "Ci-dessous vous pouvez remarquer que la classe LeNet5 a été modifiée pour y ajouter la batch normalisation."
```
Comment (mgermain) [inline tutoriaux/CNN/CNN_a_completer.ipynb:1217]: Why remove this one ?

File: tutoriaux/CNN/CNN_solutions.ipynb [hunk 2]
```diff
@@ -1244,14 +1244,14 @@
     "        super(LeNet5, self).__init__()\n",
     "        self.block1 = nn.Sequential(\n",
     "            nn.Conv2d(1, 16, kernel_size=5, padding=2),\n",
-    "            nn.BatchNorm2d(16),\n",
     "            nn.ReLU(),\n",
+    "            nn.BatchNorm2d(16),\n",
```
Comment (mgermain) [inline tutoriaux/CNN/CNN_solutions.ipynb]: According, to pytorch [source code](https://github.com/pytorch/vision/blob/master/torchvision/models/vgg.py#L71) and the [original paper](https://arxiv.org/pdf/1502.03167.pdf), batch norm goes before the activation. Why do you think it's a good idea to invert that?

File: tutoriaux/CNN/CNN_a_completer.ipynb (hunk 1 above)
Comment (hbertrand) [reply tutoriaux/CNN/CNN_a_completer.ipynb:1217]: Because there is nothing to notice, they have to implement it, not look at it.

File: tutoriaux/CNN/CNN_solutions.ipynb (hunk 2 above)
Comment (hbertrand) [reply tutoriaux/CNN/CNN_solutions.ipynb]: I remember a paper where they tried the different combinations and Relu -> BN did better. Looking for it now, I can only find [this](https://github.com/keras-team/keras/issues/1802#issuecomment-187966878). I'm not sure what the current consensus is now. It doesn't matter at all for this task though, we can leave BN -> Relu.

File: tutoriaux/CNN/CNN_a_completer.ipynb (hunk 1 above)
Comment (mgermain) [reply tutoriaux/CNN/CNN_a_completer.ipynb:1217]: True

File: tutoriaux/CNN/CNN_solutions.ipynb (hunk 2 above)
Comment (mgermain) [reply tutoriaux/CNN/CNN_solutions.ipynb]: Sure, but as it is an intro tutorial I would stick to the original paper.

**Comment by hbertrand** [comment]
I reverted to BN -> Relu

File: tutoriaux/CNN/CNN_a_completer.ipynb [hunk 3]
```diff
@@ -1200,21 +1201,20 @@
...
+    "Le *batch normalization* est une astuce qui permet, en pratique, au modèle d'apprendre plus vite. Elle agit comme régularisateur en normalisant les entrées par batch, de manière différentiable. Les données en sorties de cette couche auront une moyenne proche de 0 et une variance proche de 1.\n",
     "\n",
     "![Alt Text](https://github.com/mila-iqia/ecole_dl_mila_ivado_2018_10/blob/master/tutoriaux/CNN/images/1_Hiq-rLFGDpESpr8QNsJ1jg.png?raw=true)\n",
     "\n",
     "\n",
     "Pour plus d'informations sur le *Batch Normalization*, consultez cet [article](https://arxiv.org/pdf/1502.03167v3.pdf).\n",
     "\n",
     "### Boîte à outils\n",
-    "Pour ajouter la batch normalisation dans LeNet5, il suffit de l'ajouter parmis les couches de l'`__init__`. La classe suivante peut être utilisée:\n",
+    "Pour ajouter la batch normalisation dans LeNet5, il suffit de l'ajouter parmis les couches de l'`__init__` et de l'appeler après chaque Relu. La classe suivante peut être utilisée:\n",
```
Comment (mgermain) [inline tutoriaux/CNN/CNN_a_completer.ipynb]: You should swap this too.

File: tutoriaux/CNN/CNN_solutions.ipynb [hunk 4]
```diff
@@ -1212,21 +1213,20 @@
...
+    "Le *batch normalization* est une astuce qui permet, en pratique, au modèle d'apprendre plus vite. Elle agit comme régularisateur en normalisant les entrées par batch, de manière différentiable. Les données en sorties de cette couche auront une moyenne proche de 0 et une variance proche de 1.\n",
     "\n",
     "![Alt Text](https://github.com/mila-iqia/ecole_dl_mila_ivado_2018_10/blob/master/tutoriaux/CNN/images/1_Hiq-rLFGDpESpr8QNsJ1jg.png?raw=true)\n",
     "\n",
     "\n",
     "Pour plus d'informations sur le *Batch Normalization*, consultez cet [article](https://arxiv.org/pdf/1502.03167v3.pdf).\n",
     "\n",
     "### Boîte à outils\n",
-    "Pour ajouter la batch normalisation dans LeNet5, il suffit de l'ajouter parmis les couches de l'`__init__`. La classe suivante peut être utilisée:\n",
+    "Pour ajouter la batch normalisation dans LeNet5, il suffit de l'ajouter parmis les couches de l'`__init__` et de l'appeler après chaque Relu. La classe suivante peut être utilisée:\n",
```
Comment (mgermain) [inline tutoriaux/CNN/CNN_solutions.ipynb]: Here also

File: tutoriaux/CNN/CNN_a_completer.ipynb (hunk 3 above)
Comment (hbertrand) [reply tutoriaux/CNN/CNN_a_completer.ipynb]: Good point
